station/app/local_train_minio/LocalTrainMinIO.py

```diff
@@ -68,7 +72,7 @@ def _get_all_files_recursively(self, subfolder):
             else:
                 out_files.append(file)
 
-        return out_files
+        return out_files'''
```

Comment:
this needs to be commented back in. I.e. use same get_minio_client as before as a workaround. or we need to let david know so he knows to fix it